.travis.yml

```diff
@@ -1,7 +1,17 @@
 language: python
-python:
-  - 2.6
-  - 2.7
+python: 2.6
+env:
+  matrix:
+    - ENV_MOD_VERSION=3.2.10
+    - LMOD_VERSION=5.6.3 EASYBUILD_MODULES_TOOL=Lmod
+    - LMOD_VERSION=6.3.1 EASYBUILD_MODULES_TOOL=Lmod EASYBUILD_MODULE_SYNTAX=Lua
+matrix:
+  # mark build as finished as soon as job has failed
+  fast_finish: true
```

Comment:
sure, but maybe they won't start all at the same time, depending on the resources that are available

quick feedback in case of failing tests is useful, and it doesn't hurt